b2sdk/exception.py

```diff
@@ -199,21 +199,21 @@ def should_retry_http(self):
 
 
 class DestFileNewer(B2Error):
-    def __init__(self, dest_file, source_file, dest_prefix, source_prefix):
+    def __init__(self, dest_path, source_path, dest_prefix, source_prefix):
```

Comment:
For me, this is a compatibility breaking change as all exceptions are part of public API I guess

Reply:
Missing typing

Reply:
not according to our docs: https://b2-sdk-python.readthedocs.io/en/master/api/exception.html but i think this is an error. I guess this should be wrapped.